src/dotty/tools/dotc/typer/ProtoTypes.scala

```diff
@@ -379,6 +379,19 @@ object ProtoTypes {
     }
   }
 
+  /** Like wildApprox, but assume that all parameters of `disregarded` are not
```

Comment:
It's a bit weird to say "Like wildApprox" when this method is called wildApprox too, maybe use another name, or say, "Like the other overload of wildApprox"?